### DISCUSSION TITLE
Use Noco with a read-only mysql user [May 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2273] [Discord Threads]
> 🙋‍♂️ **cristian76**

**Comment by o1lab** [comment]
> 🙋‍♂️ **FrancyMak**  
#### Hello! Obviously with a read-only user you cannot create fields on the db, because Noco must have the ability to write and eventually modify the database.  
#  
> 🙋‍♂️ **FrancyMak**  
#### However, if you need a user to only be able to read and not edit/create, you can create a user with the viewer role.  
#  
> 🙋‍♂️ **cristian76**  
#### Yes i understand, but will be great feature if Nocodb can work with a readonly user from database and allow to create custom new fields without sending to the Database, keeps the original database untoched.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/980255328472023050)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
